wp-content/plugins/core/src/Theme/Theme_Definer.php

```diff
@@ -6,17 +6,102 @@
 use DI;
```

Comment:
Love the consolidation of all theme config into a single class. Can we see about moving webfont definitions here as well?

Reply:
Yes, fonts are on my future roadmap. They don't apply so specifically to the block editor, so I was tackling these things first.